data/BGC0001972.json

```diff
@@ -102,7 +42,19 @@
                 "molecular_formula": "C7H7N5O2"
             },
             {
-                "compound": "fervenulin"
+                "chem_struct": "CN1C2=C(C(=O)N(C1=O)C)N=CN=N2",
+                "chem_synonyms": [
+                    "Fervenulin",
```

Comment:
Maybe for a separate patch, but I don't think we need synonyms with different capitalisation